### DISCUSSION TITLE
Writing Schema Documentation

### DISCUSSION THREAD

**Discussion by laurent512** [prisma/prisma#9149] [Q&A] [answered]
Hello, 

Is there any tool to quickly build a "friendly" documentation for a schema? 

Purpose is to be allow someone to quickly grasp the purpose of each value defined in it. 
(i thought that putting some comment next to it is not so friendly)

Thank you so much

**Comment by ryands17** [comment, marked as answer]
@laurent512 👋 

You can have a look at [this](https://github.com/pantharshit00/prisma-docs-generator).

**Reply by laurent512** [reply]
so cool , thanks !

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
